pkgs/development/compilers/gcc/builder.sh

```diff
@@ -241,7 +241,7 @@ postInstall() {
     # More dependencies with the previous gcc or some libs (gccbug stores the build command line)
     rm -rf $out/bin/gccbug
 
-    if type "patchelf"; then
+    if [[ buildConfig == *"linux"* ]]; then
```

Comment:
`[[ buildConfig == *"linux"* ]]` is always false since `linux` is not a substring of the literal string `buildConfig`. Did you mean for this to be a variable or something?